tsconfig.json

```diff
@@ -17,134 +17,137 @@
 		"./typings/tsd.d.ts"
 	],
 	"files": [
-		"./src/DateObject.ts",
```

Comment:
I'm not sure why this entire array got regenerated/reordered in this commit.  Was the usual grunt task still used to generate it?

(Sorry these comments are kind of nit-picks...)